compair/api/assignment.py

```diff
@@ -150,9 +153,25 @@ def post(self, course_uuid, assignment_uuid):
                 params.get('compare_end', assignment.compare_end),
                 '%Y-%m-%dT%H:%M:%S.%fZ')
 
-        # validate answer + comparison period start & end times
+
+        assignment.enable_self_evaluation = params.get('enable_self_evaluation', assignment.enable_self_evaluation)
```

Comment:
`assignment.enable_self_evaluation = params.get('enable_self_evaluation', assignment.enable_self_evaluation)` is present here and on line 180 (please remove one)